src/features/PersonDetails/styled.js

```diff
@@ -0,0 +1,6 @@
+import styled from "styled-components";
+
+export const DetailsContainer = styled.div`
+  max-width: 1920px;
```

Comment:
I would change `max-width to 1400px`. The same max-width is in the MainPage, PersonList and MovieDetails.

I also added `padding: 0 16px` to gave some space between DetailsContainer and edge of screen